web/includes/functions.php

```diff
@@ -2129,4 +2129,37 @@ function getStreamHTML( $monitor, $scale=100 ) {
     }
 } // end function getStreamHTML
 
+function check_timezone() {
+  $system_timezone = '';
+  if (is_link('/etc/localtime')) {
```

Comment:
On freebsd, `/etc/localtime` is a file.  Generally the file is copied from `/usr/share/zoneinfo/`, not symlinked.
Perhaps use `is_file()`, as that function appears to handle both files and symlinks?